psse/psse-model/src/main/java/com/powsybl/psse/model/PsseTransformer.java

```diff
@@ -0,0 +1,618 @@
+            this.f4 = f4;
+        }
+
+        public String getVecgrp() {
+            return vecgrp;
+        }
+
+        public void setVecgrp(String vecgrp) {
+            this.vecgrp = vecgrp;
+        }
+    }
+
+    public static class SecondRecord {
+
+        @Parsed(index = 0)
+        private double r12 = 0;
+
+        @Parsed(index = 1)
+        @Validate
+        private double x12;
+
+        @Parsed(index = 2)
+        private double sbase12 = Double.NaN;
+
+        @Parsed(index = 3)
+        private double r23 = 0;
+
+        @Parsed(index = 4)
+        private double x23 = Double.NaN;
+
+        @Parsed(index = 5)
+        private double sbase23 = Double.NaN;
+
+        @Parsed(index = 6)
+        private double r31 = 0;
+
+        @Parsed(index = 7)
+        private double x31 = Double.NaN;
+
+        @Parsed(index = 8)
+        private double sbase31 = Double.NaN;
+
+        @Parsed(index = 9)
+        private double vmstar = 1;
+
+        @Parsed(index = 10)
+        private double anstar = 0;
+
+        public double getR12() {
+            return r12;
+        }
+
+        public void setR12(double r12) {
+            this.r12 = r12;
+        }
+
+        public double getX12() {
+            return x12;
+        }
+
+        public void setX12(double x12) {
+            this.x12 = x12;
+        }
+
+        public double getSbase12() {
+            return sbase12;
+        }
+
+        public void setSbase12(double sbase12) {
+            this.sbase12 = sbase12;
+        }
+
+        public double getR23() {
+            return r23;
+        }
+
+        public void setR23(double r23) {
+            this.r23 = r23;
+        }
+
+        public double getX23() {
+            return x23;
+        }
+
+        public void setX23(double x23) {
+            this.x23 = x23;
+        }
+
+        public double getSbase23() {
+            return sbase23;
+        }
+
+        public void setSbase23(double sbase23) {
+            this.sbase23 = sbase23;
+        }
+
+        public double getR31() {
+            return r31;
+        }
+
+        public void setR31(double r31) {
+            this.r31 = r31;
+        }
+
+        public double getX31() {
+            return x31;
+        }
+
+        public void setX31(double x31) {
+            this.x31 = x31;
+        }
+
+        public double getSbase31() {
+            return sbase31;
+        }
+
+        public void setSbase31(double sbase31) {
+            this.sbase31 = sbase31;
+        }
+
+        public double getVmstar() {
+            return vmstar;
+        }
+
+        public void setVmstar(double vmstar) {
+            this.vmstar = vmstar;
+        }
+
+        public double getAnstar() {
+            return anstar;
+        }
+
+        public void setAnstar(double anstar) {
+            this.anstar = anstar;
+        }
+    }
+
+    public static class ThirdRecord {
+
+        @Parsed(index = 0)
+        private double windv = Double.NaN;
```

Comment:
Here too the default value setting is more complex and need linked between objects. You put Nan to remember that a more complex treatement has to be done ?

Reply:
At the end of all the records ?